<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="8,1,1">
     <item>
      <widget class="QDial" name="pDial">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3,2,3">
       <item>
        <widget class="QLabel" name="pLedLabel">
         <property name="text">
          <string>led</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pLedValLabel">
         <property name="text">
          <string>led_val</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pKeyLabel">
         <property name="text">
          <string>key</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pKeyValLabel">
         <property name="text">
          <string>key_val</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="pProgressBar">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
